cmd/zpool/zpool_main.c

```diff
@@ -7253,15 +7253,71 @@ print_error_log(zpool_handle_t *zhp)
 	elem = NULL;
 	while ((elem = nvlist_next_nvpair(nverrlist, elem)) != NULL) {
 		nvlist_t *nv;
-		uint64_t dsobj, obj;
-
-		verify(nvpair_value_nvlist(elem, &nv) == 0);
-		verify(nvlist_lookup_uint64(nv, ZPOOL_ERR_DATASET,
-		    &dsobj) == 0);
-		verify(nvlist_lookup_uint64(nv, ZPOOL_ERR_OBJECT,
-		    &obj) == 0);
+		uint64_t dsobj, obj, data_block_size, indirect_block_size;
+		uint64_t *block_ids;
+		int64_t *indirect_levels;
+		unsigned int error_count_blkids;
+		unsigned int error_count_levels;
+
+
+		nv = fnvpair_value_nvlist(elem);
```

Comment:
My personal preference for things like this is to declare them on the same line you assign them for ease of reading, but that may just be me.